### ISSUE TITLE
False positive: no-self-argument on abstract static methods

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#316]
Originally reported by: **BitBucket: [ahammel](http://bitbucket.org/ahammel), GitHub: @ahammel?**

---

Pylint incorrectly thinks that the first argument of an abstract static method should be 'self':

```
#!python

from abc import ABCMeta, abstractstaticmethod

class SillyExample(object, metaclass=ABCMeta):
    @staticmethod
    def i_dont_need_self(foo, bar):
        # Does not raise E0213 (correct)
        pass

    @abstractstaticmethod
    def i_dont_need_self_either(foo, bar):
        # Raises E0213 (incorrect)
        pass
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/316
